scripts/generate-vectors.ts

```diff
@@ -8,9 +8,13 @@ import {
   ed25519,
   MAX_BIP_32_INDEX,
   secp256k1,
+  ed25519Bip32,
 } from '../src';
-import type { Curve } from '../src/curves';
-import { createBip39KeyFromSeed } from '../src/derivers/bip39';
+import { type Curve } from '../src/curves';
+import {
+  entropyToCip3IcarusRootKeyPair,
```

Comment:
I see we refer to this "root keypair" also as "master key(pair)" in other places (see `masterKeyGenerationSpec`). I'd suggest unifying the naming to "master" keypair